src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/DefaultFallbackCapabilitiesFilterResolver.cs

```diff
@@ -60,8 +60,6 @@ internal class DefaultFallbackCapabilitiesFilterResolver : FallbackCapabilitiesF
                 case Methods.TextDocumentDocumentHighlightName:
                     return CheckHighlightCapabilities;
                 case "textDocument/semanticTokens":
-                case Methods.TextDocumentSemanticTokensFullName:
```

Comment:
We should keep these since they're in the contained language stack

Reply:
Definitely! WebTools and .NET Interactive also utilize the ContainedLanguage library to power HTML/.NET Interactive experiences; meaning, if they send semantic token messages via our LSP request invoker this call path is hit and we ensure to associate the correct capability filter for them

Reply:
Ahh interesting, thanks for the explanation!